src/main/java/org/folio/inventory/resources/InventoryConfiguration.java

```diff
@@ -0,0 +1,21 @@
+package org.folio.inventory.resources;
+
+import com.google.common.collect.Sets;
+import java.util.Set;
+
+/**
+ * Stores configuration properties for Inventory entities: Instances, Items, Holdings
+ */
+public class InventoryConfiguration {
+  protected static final Set<String> BLOCKED_FIELDS = Sets.newHashSet(
```

Comment:
Why is this protected, there does not appear to be any inheritance?

Reply:
Sonar advised me to use 'protected' access instead of 'public' , because of this field is used within the class in the same package.

Reply:
It seems Sonar gets very unhappy with this being public. I suggest we put this back to protected, apologies :-/